src/scanspec/regions.py

```diff
@@ -32,6 +33,8 @@
     "find_regions",
 ]
 
+NpMask = npt.NDArray[np.bool]
```

Comment:
BTW I recently discovered you can specify shape too, although most of numpy returns `shape=Any` so it's of limited use at the mo:
```python
ScalarType_co = TypeVar("ScalarType_co", bound=np.generic, covariant=True)
OneDArray = np.ndarray[tuple[int], np.dtype[ScalarType_co]]
TwoDArray = np.ndarray[tuple[int, int], np.dtype[ScalarType_co]]

NPMask = OneDArray[np.bool]
```

Reply:
Interesting... worth a separate issue to specify shape where possible?